### PR TITLE
@ashfurrow => Initial work on beginners lesson 2

### PR DESCRIPTION
**Pull request by orta**
I started on `Enum`s, but realized there was still a lot of basics to cover before I could do too much. I might put it in at the end as an evolution of the `Bool` tomorrow.

### CODE REVIEW

File: Beginners/Lesson Two/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+So what we're looking at, in plain English: _if something is exciting, show a party popper else recommend a good youtube video_. We can use an `if` and an ` else` statement to represent a very large amount of logic. `if`s have one more thing, that I'd like to go over. Before we do that, we should consider what happens if Andrew WK is in the room.
+
+``` swift
+var isAndrewWKAround = false
+```
+
+Well, he would have an important impact on everyone's exciting-ness. Let's try and represent that in our if statement.
+
+``` swift
+# Whoah, Andrew WK?
```
Comment (ashfurrow) [inline Beginners/Lesson Two/README.md]: `#`?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Pre-requisites: Yosemite, Xcode 7
 
 * Beginners
  1. [Variables - Strings - If - Loops](Beginners/Lesson One)
- 1. Intro to types / structs/ enums / classes / tuples
```
Comment (ashfurrow) [inline README.md:11]: Ah yeah, think I added that by accident.

File: Beginners/Lesson Two/README.md (hunk 1 above)
Comment (orta) [inline Beginners/Lesson Two/README.md]: Given the large amount of "repeating" over the same code path, I wanted to be able to ensure there is an easy way to differentiate in the playground as will show a lot of repeated code bits

Comment (ashfurrow) [inline Beginners/Lesson Two/README.md]: Makes sense, but within a swift code black, shouldn't `#` be `//`?

Comment (orta) [inline Beginners/Lesson Two/README.md]: oh yeah, its all looks like ruby so much!

**Comment by orta** [comment]
Alright, updated :+1:
